dll/shellext/fontext/CFontCache.cpp

```diff
@@ -62,7 +66,56 @@ const CStringW& CFontInfo::File()
     return m_File;
 }
 
+void CFontInfo::ReadAttrs()
+{
+    CStringW File = g_FontCache->Filename(this, true);
+
+    m_AttrsRead = true;
+
+    WIN32_FIND_DATAW findFileData;
+    HANDLE hFile = FindFirstFileW(File, &findFileData);
+    if (hFile != INVALID_HANDLE_VALUE)
```

Comment:
Or maybe, just return now if `hFile == INVALID_HANDLE_VALUE`, and do the rest below following: this will avoid that extra level of indentation.

Reply:
'that extra level of indentation' would be a good reason if it was 4 or 5 levels deep, not when it is the only indentation.